subprojects/model-core/src/test/groovy/org/gradle/internal/instantiation/AsmBackedClassGeneratedManagedStateTest.groovy

```diff
@@ -27,25 +27,7 @@ import org.junit.ClassRule
 import spock.lang.Shared
 import spock.lang.Unroll
 
-import static org.gradle.internal.instantiation.AsmBackedClassGeneratorTest.AbstractBean
-import static org.gradle.internal.instantiation.AsmBackedClassGeneratorTest.AbstractBeanWithInheritedFields
-import static org.gradle.internal.instantiation.AsmBackedClassGeneratorTest.AbstractClassWithTypeParamProperty
-import static org.gradle.internal.instantiation.AsmBackedClassGeneratorTest.Bean
-import static org.gradle.internal.instantiation.AsmBackedClassGeneratorTest.BeanWithAbstractProperty
-import static org.gradle.internal.instantiation.AsmBackedClassGeneratorTest.InterfaceBean
-import static org.gradle.internal.instantiation.AsmBackedClassGeneratorTest.InterfaceDirectoryPropertyBean
-import static org.gradle.internal.instantiation.AsmBackedClassGeneratorTest.InterfaceFileCollectionBean
-import static org.gradle.internal.instantiation.AsmBackedClassGeneratorTest.InterfaceFilePropertyBean
-import static org.gradle.internal.instantiation.AsmBackedClassGeneratorTest.InterfaceListPropertyBean
-import static org.gradle.internal.instantiation.AsmBackedClassGeneratorTest.InterfaceMapPropertyBean
-import static org.gradle.internal.instantiation.AsmBackedClassGeneratorTest.InterfaceNestedBean
-import static org.gradle.internal.instantiation.AsmBackedClassGeneratorTest.InterfacePrimitiveBean
-import static org.gradle.internal.instantiation.AsmBackedClassGeneratorTest.InterfacePropertyBean
-import static org.gradle.internal.instantiation.AsmBackedClassGeneratorTest.InterfacePropertyWithParamTypeBean
-import static org.gradle.internal.instantiation.AsmBackedClassGeneratorTest.InterfaceSetPropertyBean
-import static org.gradle.internal.instantiation.AsmBackedClassGeneratorTest.InterfaceWithDefaultMethods
-import static org.gradle.internal.instantiation.AsmBackedClassGeneratorTest.Param
-
+import static org.gradle.internal.instantiation.AsmBackedClassGeneratorTest.*
```

Comment:
Was this intentional?

Reply:
On `master` someone changed the imports, so I got a merge conflict. I restorted the `*` import for the test to have the same state as on `master`. Shall I expand the `*` import again?

Reply:
It's fine, just wanted to understand if this was intentional, thanks.